src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -600,5 +604,33 @@ private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken r
 
             return refKind;
         }
+
+        internal static void AddNullCheckingErrorsToParameter(DiagnosticBag diagnostics, ParameterSymbol parameter, bool isLambdaOrLocalFunction)
```

Comment:
>internal [](start = 8, length = 8)

Please test each of these diagnostics (including missing `System.ArgumentException`) with local functions.